fares-data-build-tool/fdbt-site
# feat: added matching json for multiOperatorGeoZone

# Description

-   This PR adds the data collected from the user about multi operator geoZone tickets.

# Testing instructions

-   Pull down this branch and run the site locally. Validate that the matching json produced at the end of the multi operator, geoZone ticket flow is the correct format (check this against the mock data in the fdbt-dev repo).
-   Validate that the other ticket flows still produce matching json.

# Type of change


# Checklist:
